indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -220,18 +224,66 @@
 {% endmacro %}
 
 
-{% macro _render_review_trigger(abstract) %}
-    {% set tracks = abstract.get_reviewed_for_tracks_by_user(session.user, include_reviewed=true)|sort(attribute='title') %}
-    {% set reviews = abstract.get_reviews(user=session.user) %}
-    {% set reviewed = reviews|map(attribute='track') %}
+{% macro _render_input_no_comment(abstract, review_form=none) %}
+    {% set reviewer_data = abstract.get_reviewer_render_data(session.user) %}
+    {% if reviewer_data.tracks|length == 1 and not reviewer_data.reviews %}
+        {{ render_review_form(review_form, abstract, reviewer_data.tracks|first) }}
+    {% else %}
+        {% set missing_tracks = reviewer_data.missing_tracks|list|length %}
```

Comment:
no need for `|list` anymore, and i'd just use `reviewer_data.missing_tracks|length` below and get rid of this line. if not be consistent and use `missing_tracks` in the next line